imageregistry/v1/00_imageregistry.crd.yaml

```diff
@@ -1296,6 +1296,22 @@ spec:
                           storage services. Optional, defaults based on the Region
                           that is provided.
                         type: string
+                      trustedCA:
+                        description: "trustedCA is a reference to a ConfigMap containing
+                          a CA certificate bundle that should be trusted when establishing
+                          a connection to S3. It is used merged with the system default
+                          trust bundle. \n The namespace for the ConfigMap referenced
+                          by trustedCA is \"openshift-config\". The key for the bundle
```

Comment:
Are we going to read this from the `openshift-config` namespace ? Why not from the image-registry namespace ?

Reply:
`openshift-config` is the namespace where all user provided configs should be. The image registry operator is kinda special, it has a strange CRD name, and occasionally it expects configmaps/secrets in its own namespace.

@deads2k do we have documented policy about `openshift-config`?